### ISSUE TITLE
Rename CollectionItem too hard to find

### ISSUE THREAD

**Issue by mgmhunt** [brunomikoski/ScriptableObjectCollection#146]
Hi, I can't see anyway to easily rename the SO after they have been created - as per when you first create them and the object name is highlighted and editable.

EDIT - Found it F2 after looking through code - too hidden for me (on a Mac too so not instinctual!)

Added Rename Asset to ShowOptionsForIndex line 724 of CollectionCustomEditor

```
            menu.AddSeparator("");
            menu.AddItem(
                new GUIContent("Rename Asset"),
                false,
                () => { RenameItemAtIndex(targetIndex); }
            );
```

**Comment by brunomikoski** [comment]
Feel free to open a pull request with this change! But just FIY you can also just rename the asset itself!

**Comment by mgmhunt** [comment]
Yep will try to get round to it.  Renaming the asset via Project view was the long way around but the right click is working great - I've also got it renaming nested scriptable objects which can't be done via the Project view.

**Comment by brunomikoski** [comment]
Added your suggestion https://github.com/brunomikoski/ScriptableObjectCollection/releases/tag/v2.3.7
